codegenerator/cli/templates/static/codegen/src/globalState/GlobalState.res

```diff
@@ -729,6 +729,7 @@ let injectedTaskReducer = (
       blockTimestampThreshold,
       nextEndOfBlockRangeScannedData,
     }) =>
+    let timeRef = Hrtime.makeTimer()
```

Comment:
Won't it be a problem having it outside of `Env.saveBenchmarkData` check?

Reply:
It's just a variable with with a value for time. It would hardly affect performance.